groups/managers.py

```diff
@@ -3,10 +3,11 @@
 from django.apps import apps
 from django.contrib.auth import get_user_model
 from django.db import models
-from polymorphic import PolymorphicManager, PolymorphicQuerySet
+from polymorphic.managers import PolymorphicManager, PolymorphicQuerySet
 
 
-DEFAULT_WITHIN_DAYS = apps.get_app_config('groups').default_within_days
+def default_within_days():
```

Comment:
I guess this is no longer necessary. I think the error was always the same, that `groups` was not in installed apps.